addons/calendar/models/calendar_contact.py

```diff
@@ -11,6 +11,8 @@ class Contacts(models.Model):
     user_id = fields.Many2one('res.users', 'Me', required=True, default=lambda self: self.env.user)
     partner_id = fields.Many2one('res.partner', 'Employee', required=True)
     active = fields.Boolean('Active', default=True)
+    partner_checked = fields.Boolean('Checked', default=True,
```

Comment:
What is the purpose of "active" field on contact model now that filter status is stored using partner_checked ? Do we still need it ?